### ISSUE TITLE
Color Palette Synchronization

### ISSUE THREAD

**Issue by fantasai** [w3c/w3c-website#208]
**Describe the issue**
Nobody ever responded to https://lists.w3.org/Archives/Public/public-website-redesign/2020Oct/0000.html in the three years since it was posted.

**Comment by koalie** [comment]
tl;dr: 
This is out of [scope](https://www.w3.org/2019/11/website-redesign-rfp.html#scope) for the redesign which only focused on the '/TR homepage', not (yet) how specs themselves are designed. We should assess later if it's desirable for the colors to match.

longer response:
I'm pretty sure I had responded then, but I can't find any evidence that I did, so it's likely I told you directly instead. In any case, apologies and thanks for raising this again!

Studio 24 did look at the colors we use elsewhere, but I don't remember if they told us anything regarding other colors when they developed the [color palette](https://design-system.w3.org/settings/#color-palette).
